tests/pkg_relic/Makefile

```
@@ -0,0 +1,46 @@
+include ../Makefile.tests_common
+
```

This one should also be enable for CI

```suggestion

TEST_ON_CI_WHITELIST += all
```

But as `arm` was not supported before maybe it should just be `+= native`.